### PR TITLE
New flags added to Application Insights Component

### PR DESCRIPTION
**Pull request by dhaiman**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [x] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [ ] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by zcxcasd** [comment]
> {

next time, the first iteration should always be a copy of the previous version into the new version file without any changes.
Without this, it makes it very difficult for any changes to be diffed and reviewed. If you want a faster review, please abandon and recreate. 

---
Refers to: specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-02-preview/components_API.json:1 in 8c74a96. [](commit_id = 8c74a96b05380108f5790c8b366d7961029f7ef4, deletion_comment = False)

**Comment by dhaiman** [comment]
These two new properties are nullable boolean and will not affect old users.

**Comment by kairu-ms** [comment]
Please fix CI issues.

**Comment by dhaiman** [comment]
CI fixed except BreakingChange, as I mentioned these properties are nullable properties added to the preview API version

**Review by kairu-ms — APPROVED** [review]

**Comment by kairu-ms** [comment]
> CI fixed except BreakingChange, as I mentioned these properties are nullable properties added to the preview API version

It needs breaking change review if you want to add them in existing API-version. Please create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

File: specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-02-preview/components_API.json [hunk 1]
```diff
@@ -601,6 +601,14 @@
             "name": "IngestionMode",
             "modelAsString": true
           }
+        },
+        "DisableNonAADAuth": {
```
Comment (olegsych) [inline specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-02-preview/components_API.json:605]: Shouldn't `DisableNonAADAuth` be `DisableLocalAuth`?

Comment (dhaiman) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-02-preview/components_API.json]: fixed

**Review by kairu-ms — APPROVED** [review]

**Comment by chiragg4u** [comment]
ARM API review is not required for this PR, please follow the process given @ https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review

**Review by JeffreyRichter — APPROVED** [review]

**Pull request merged**